assets/dev/js/editor/controls/date-time.js

```diff
@@ -10,6 +10,32 @@ export default class extends ControlBaseDataView {
 		this.ui.input.flatpickr( options );
 	}
 
+	onBaseInputChange() {
+		super.onBaseInputChange( ...arguments );
+
+		if ( ! this.model.attributes?.validation ) {
+			return;
+		}
+
+		if ( this.model.attributes?.validation.minDate ) {
+			const controlName = this.model.attributes.validation.minDate.control_name;
```

Comment:
```JS
const { control_name: controlName, operator } = this.model.attributes.validation.minDate;
```

Maybe? (and then remove the `operator` variable in line 31)

Reply:
Done